src/jobs/models/jobManager.js

```diff
@@ -8,7 +8,8 @@ const logger = require('../../common/logger'),
     databaseConnector = require('./database/databaseConnector');
 
 let cronJobs = {};
-const JOB_PLATFORM_NAME = 'predator.%s';
+const PREDATOR_RUNNER_PREFIX = 'predator';
```

Comment:
In my opinion `PREDATOR_RUNNER_PREFIX` isn't really used (also in the other functions) and can be removed